repository/repository/targets.json

```diff
@@ -104,6 +96,13 @@
 				},
 				"length": 740
 			},
+			"rekor.0.pub": {
```

Comment:
This doesn't look correct, the Rekor delegation should sign off on the rekor shard key - Are we ok with this for now?

Reply:
This was also present in the v3 root signing.

It's going to be a lot of work to re-sign the targets at this point. I think we should just resolve this with the next root signing.